samza-core/src/main/java/org/apache/samza/zk/ZkKeyBuilder.java

```diff
@@ -27,6 +27,7 @@
  *   - /
  *      |- groupId/
  *          |- JobModelGeneration/
+ *              |- activeJobModelVersion (data contains the most recent active job model version)
```

Comment:
Semantically what is the difference between activeJobModelVersion & JobModelVersion.? On what scenarios would both of them would be different?

Reply:
I had this documented in the getter of the `ZkKeyBuilder` in the same class. Here is the difference

```
Denotes the path where the most recent active job model version is stored. The version of the job model is the
   * most recent agreed upon version by the quorum. It differs from the <i>jobModelVersion</i> path which may
   * have a newer version of job model published by the leader in during rebalance before consensus is achieved.
   * @return the path where most recent active job model is stored
   ```